www/src/app/services/alfred-api-service.js

```diff
@@ -0,0 +1,30 @@
+import { API_URL } from '../../config';
+import { get, post } from '../utils';
+
+
+export function fetchUpdatesByAuthor(author) {
```

Comment:
I think we'll also need a service function of getting updates for a given report. See `www/src/app/views/ReportContainer`.